extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -580,7 +589,7 @@ private function getOtherLanguages( Title $title ) {
 		}
 
 		$url = $title->getFullURL();
-		//$url = str_replace( '.rychu.wikia-dev.com', '.wikia.com', $url );
+		// $url = str_replace( '.rychu.wikia-dev.com', '.wikia.com', $url );
```

Comment:
:see_no_evil: I still don't get it, why it's left here. Is it read by some machine/bot/whatever? If not, there's not much difference between uncommenting this thing and just pasting it on dev-/sandbox.